css/style.css

```diff
@@ -0,0 +1,402 @@
+* {
+  padding: 0;
+  margin: 0;
+}
+
+body {
+  font-family: system-ui, "Segoe UI", Roboto, Oxygen, Ubuntu, Cantarell, "Open Sans", "Helvetica Neue", sans-serif;
+  background-color: #f9f9f9;
+}
+
+nav {
+  display: none;
+}
+
+a {
+  text-decoration: none;
+  color: #036bd3;
+}
+
+button {
+  margin-top: 0.5rem;
+  height: 3rem;
+  width: 11rem;
+  border: 1px solid #036bd3;
+  color: #036bd3;
+  font-size: 10px;
+  border-radius: 0.8rem;
+  cursor: pointer;
+  background-color: transparent;
+}
+
+header {
+  background-image: linear-gradient(115deg, #1a1a1a80, #0d0d0e70), url("../assets/her-one.png");
```

Comment:
- [ ] background should have a transparency ratio. 
- [ ] text justify,  font-style bold
- [ ] increase the line height

Reply:
i don't think i understand the transparency ratio part

Reply:
look at the original design